pkg/k8shandler/common.go

```diff
@@ -213,6 +213,17 @@ func newProxyContainer(imageName, clusterName string) (v1.Container, error) {
 	if err != nil {
 		return v1.Container{}, err
 	}
+
+	cpuLimit, err := resource.ParseQuantity("100m")
```

Comment:
can you override these values in the CR?

Reply:
looks like we don't have a field for this to be set in the CR

Reply:
And we don't want to - us or the auth team should provide reasonable values. Users of openshift logging shouldn't face micro-decisions like this. 

How would the CR evolve over time? For example if we remove the proxy?